sdks/python/apache_beam/runners/common.py

```diff
@@ -477,8 +482,20 @@ def __init__(self,
 
     # Optimize for the common case.
     main_receivers = tagged_receivers[None]
+
+    # TODO[BEAM-3937]:Remove if block after per-element-output-counter released.
+    experiments = RuntimeValueProvider.get_value('experiments', str, [])
+    per_element_output_counter = None
+    if 'outputs_per_element_counter' in experiments:
+      per_element_output_counter_name = (
+          CounterName('per-element-output-count-meta',
+                      step_name=operation_name))
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Why is the step name the operation name and not the step name?  Please add a clarifying comment for why, if this needs to be the case.